internal/gtou/gtou.go

```diff
@@ -95,6 +100,13 @@ func (c *Converter) newDocument(inv *bill.Invoice) error {
 	return nil
 }
 
+func getTypeCode(inv *bill.Invoice) (string, error) {
+	if inv.Tax == nil || inv.Tax.Ext == nil || inv.Tax.Ext[untdid.ExtKeyDocumentType].String() == "" {
+		return "", fmt.Errorf("invoice must contain tax extensions")
```

Comment:
I'd recommend using a more explicit error message here with nested validation, like:
```suggestion
		return "", validation.Errors{
			"tax": validation.Errors{
				"ext": {
					untdid.ExtKeyDocumentType.String(): "required",
				},
			},
		}
```